src/View/AbstractDirectoryView.vala

```diff
@@ -939,19 +939,36 @@ namespace FM {
 
             /* Block the async directory file monitor to avoid generating unwanted "add-file" events */
             slot.directory.block_monitor ();
-            Marlin.FileOperations.new_file (this as Gtk.Widget,
-                                            null,
-                                            parent_uri,
-                                            null,
-                                            null,
-                                            0,
-                                            create_file_done);
+            Marlin.FileOperations.new_file (
```

Comment:
`.begin` missing?

Reply:
oops, indeed, it was written with the ton of warnings :)